src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/NewCohortWithSingleApprenticeResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.CommitmentsV2.Api.Types
+{
+    // Do we need to return encoded values and should we include the AccountId? 
+    public sealed class NewCohortWithSingleApprenticeResponse
+    {
+        public int Cohortid { get; set; }
```

Comment:
This is probably better as a "CreateCohortResponse"
CohortId should be a long (and have a capital I)
Could we also have the cohort reference as well as the api will have to generate it, so it may as well return it
DraftApprenticeshipId is probably not required (but would be a long if it was :-) )